udata_front/theme/gouvfr/assets/js/api/datasets.ts

```diff
@@ -42,3 +42,21 @@ export function getFrequenciesUrl () {
 export function getSpatialGranularities () {
   return api.get<Array<SpatialGranularity>>(getLocalizedUrl("spatial/granularities/")).then(resp => resp.data);
 }
+
+export type GetOrganizationDatasetsData = {
+  data: Array<Dataset>;
+  next_page: string | null;
+  page: number;
+  page_size: number;
+  total: number;
+};
```

Comment:
We don't have a type `Paginator<T>` somewhere?

Reply:
I'm taking it from https://github.com/datagouv/udata-front/pull/446